davisinteractive/session/session_test.py

```diff
@@ -268,11 +268,11 @@ def test_integration_multiple_sequences(self, mock_davis):
         self.assertTrue('auc' in summary)
         self.assertTrue('metric_at_threshold' in summary)
         self.assertEqual(summary['metric_at_threshold']['threshold'], 60)
-        np.testing.assert_almost_equal(summary['metric_at_threshold']['metric'],
+        np.testing.assert_almost_equal(summary['metric_at_threshold']['J_an_F'],
```

Comment:
Test before submitting ;) Should be `J_and_F`. This should fix the tests.

Reply:
Yes, it should be fixed now